.circleci/config.yml

```diff
@@ -736,16 +871,6 @@ jobs:
             cd priv/cabbage
             mix test
 
-  publish_child_chain:
```

Comment:
we're removing image publishing and deploying from this repo because we have grandfather.